lib/matplotlib/backends/backend_agg.py

```diff
@@ -485,6 +485,42 @@ def print_raw(self, filename_or_obj, *args, **kwargs):
     print_rgba = print_raw
 
     def print_png(self, filename_or_obj, *args, **kwargs):
+        """
+        Write the png data to the given file.
+
+        Parameters
+        ----------
+        filename_or_obj : str or PathLike or file-like object
+            The file to write to.
+
+        metadata : dict, optional
+            Metadata in the PNG file as key-value pairs of bytes or latin-1 
+            encodable strings.
+            According to the PNG specification, keys must be shorter than 79
+            chars.
+
+            The PNG specification defines some common keywords that may be
```

Comment:
This is quite a long list, it might be worth linking to the relevant part of the PNG specification instead?

Reply:
Thanks for the hint. I've added a link to the PNG specification. Nevertheless, I think it's good to have a list of the common keys here so that people don't have to check the external documentation just to get an idea how this is typically used.